lib/foreman_ansible/register.rb

```diff
@@ -82,7 +82,7 @@
         :edit_ansible_variables, :destroy_ansible_variables]
 
   role 'Ansible Tower Inventory Reader',
-       [:view_hosts, :view_hostgroups, :view_facts],
+       [:view_hosts, :view_hostgroups, :view_facts, :view_schedule],
```

Comment:
Do we need separate view_schedule permission? I think this should be the same permission as for genertimg regular report

Reply:
I've reused `generate_report_tempaltes` permission if it makes sense.

Reply:
yes, that's exactly what I had in mind